pkgs/development/libraries/libtiff/default.nix

```diff
@@ -38,9 +38,8 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ libdeflate ]; # TODO: move all propagatedBuildInputs to buildInputs.
 
-  enableParallelBuilding = true;
-
-  doCheck = true;
+  doInstallCheck = true;
```

Comment:
Does autoconf also build in parallel by default? Or is this a mistake?

Reply:
Ah, this must have been caused by my rebase from master to staging.